tests/test_resources.py

```diff
@@ -49,5 +49,77 @@ def test_httpverb(self):
             response = method('foo')
             assert response.success
 
+    def test_pagination(self):
+      self.client.url = 'mock://api.com/{param}{?per_page}'
```

Comment:
@pengwynn Any insight on this? Can only the paginated resources that advertise 'per_page` have the parameter attached to the query?